_includes/manuals/nightly/2.1_quickstart_installation.md

```diff
@@ -60,7 +60,7 @@ sudo yum-config-manager --enable rhel-7-server-optional-rpms rhel-server-rhscl-7
 
   <p>
     Using a recent version of Puppet is recommended, which is available from the Puppet Labs repository.
-    To use Puppet 6.x with Puppet Agent and Puppet Server:
+    To use Puppet 6.x with Puppet Agent and Puppet server:
```

Comment:
It does look odd to write `Puppet Agent` but then `Puppet server` in the same sentence. Should we go with `Puppet agent` as well then?